### DISCUSSION TITLE
thanhle.blog/blog/giao-tiep-hieu-qua-cho-dev

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#43] [Ideas]
# thanhle.blog/blog/giao-tiep-hieu-qua-cho-dev

Cách mình cải thiện giao tiếp khi làm dev

https://thanhle.blog/blog/giao-tiep-hieu-qua-cho-dev

**Comment by thinhngtruong** [comment]
Em vừa đọc xong bài viết của a Huy Trần bên Kipalog cũng nói về vấn đề hỏi đáp thì gặp bài viết này, cảm ơn anh vì bài viết chất lượng và ví dụ cụ thể. :D 

https://kipalog.com/posts/Tu-hoc-lap-trinh---Hoi-lam-sao-de-duoc-tra-loi

**Reply by thanhlmm** [reply]
Thanks bạn, vui vì nó có ích. Mình cũng hay đọc blog của anh Huy TD ở https://snacky.blog/

**Comment by nguyenhoaikhang37** [comment]
Bài viết thực sự bổ ích anh ơi, mong anh làm 1 bài viết các npm thường dùng trong dự án React (như trong bài viết có đề cập 2 thằng là keen-slide và splidejs để làm carousel), hoặc cho em hỏi các npm hay dùng thường kiếm ở đâu ạ

**Reply by thanhlmm** [reply]
anh cũng search google thôi, nói chung cũng khá gian nan. Anh cũng có một số trick để tìm cho nhanh, để khi nào anh thử tổng hợp lại xem đủ một bài ko nhé

**Reply by nguyenhoaikhang37** [reply]
dạ rất hóng chủ đề này luôn ạ, em cũng search nhiều keyword nhưng ít khi nào kiếm đc npm nào xịn với nhiều lượt tải T.T

**Comment by thanhlmm** [comment]
With the global adoption of the Internet, gurus suddenly are as close as your Enter key. 
So, how do you find one, and how do you get one to talk with you?

We find there are some simple tricks.

- Know exactly what you want to ask, and be as specific as you can be.
- Frame your question carefully and politely. Remember that you're asking a favor; 
  don't seem to be demanding an answer.
- Once you've framed your questioned, stop and look again for the answer. 
  Pick out some keywords and search the web. Look for appropriate FAQs (lists of 
  frequently asked questions with answers).
- Decide if you want to ask publicly or privately. Usenet news-groups are wonderful 
  meeting places for experts on just about any topic, but some people are wary of these 
  groups' public nature. Alternatively, you can always e-mail your guru directly. Either 
  way, use a meaningful subject line. (" Need Help!!! " doesn't cut it.)
- Sit back and be patient. People are busy, and it may take days to get a specific answer.

Finally, please be sure to thank anyone who responds you. And if you see people asking 
questions you can answer, play your part and participate.

---

Quotes khá hay mình lượm dc ở đây: https://kipalog.com/posts/Tu-hoc-lap-trinh---Hoi-lam-sao-de-duoc-tra-loi

**Comment by binhnguyen2501** [comment]
<3

**Comment by nangmuathu** [comment]
Cám ơn anh đã chia sẻ. Hay tóa <3
